tests/Doctrine/Tests/DBAL/Platforms/AbstractSQLServerPlatformTestCase.php

```diff
@@ -508,6 +508,24 @@ public function testCreateClusteredIndex()
         self::assertEquals('CREATE CLUSTERED INDEX idx ON tbl (id)', $this->_platform->getCreateIndexSQL($idx, 'tbl'));
     }
 
+    public function testFailureToCreateClusteredUnnamedNullIndex()
```

Comment:
I believe for backwards compatibility, there could be `null` or `''` (blank string). I would suggest we handle both until a new major version changes the code - otherwise you decide on best fit @morozov.